api/src/test/java/org/openmrs/notification/db/hibernate/HibernateAlertDAOTest.java

```diff
@@ -60,13 +60,13 @@ public void deleteAlert_shouldReturnNullAfterDeleting() {
 	
 	@Test
 	public void getAllAlerts_shouldReturnOnlyNonExpiredAllerts() {
-		Assertions.assertEquals(hibernateAlertDAO.getAllAlerts(false).size(), 1);
+		Assertions.assertEquals(hibernateAlertDAO.getAllAlerts(false).size(), 0);
 	}
 	
 	@Test
 	public void getAlerts_shouldReturnAllAlertsWhenUserIsSpecified() {
 		User user = Context.getUserService().getUserByUuid("c1d8f5c2-e131-11de-babe-001e378eb77e");
-		Assertions.assertEquals(hibernateAlertDAO.getAlerts(user, true, false).size(), 1);
+		Assertions.assertEquals(hibernateAlertDAO.getAlerts(user, true, false).size(), 0);
```

Comment:
What does changing this to "0" do? I assume we don't want to lose the alert notification (but hey, I don't know how to review code, so feel free to ignore this comment)

Reply:
@gracepotma that is a very nice catch! 👍

Reply:
@gracepotma , thats an excellent code review , So your comment is very awesome as @dkayiwa  said its a nice catch